tests/unit/WDCMethods.spec.js

```diff
@@ -5,7 +5,8 @@ import {
   getTimeSeriesByID,
   reformatTimeString,
   sanitizeVariableName,
-  generateDateTime
+  generateDateTime,
+  generateMultiURL
```

Comment:
I would also like to see a test for the `combineJSONList` function.

Reply:
sounds good, I'll create one